src/components/calcite-input/calcite-input.tsx

```diff
@@ -283,7 +283,9 @@ export class CalciteInput implements LabelableComponent, FormComponent {
     this.scale = getElementProp(this.el, "scale", this.scale);
     this.status = getElementProp(this.el, "status", this.status);
     this.inlineEditableEl = this.el.closest("calcite-inline-editable");
-    this.editingEnabled = this.inlineEditableEl?.editingEnabled;
```

Comment:
cc @eriklharper this was messing up tests. It was setting editingEnabled to undefined always which was undoing the default false value. Lets be careful with connectedCallback stuff.